docs/backend/control-panels.md

```diff
@@ -42,4 +42,8 @@ To manually register a view as a control panel, add the following registration t
           <permission>Manage portal</permission>
     </configlet>
   </object>
-```
\ No newline at end of file
+```
+
+```{seealso}
+See the chapter {ref}`training:controlpanel-label` from the Mastering Plone 6 Training
```

Comment:
This link is broken. See rendered:

https://deploy-preview-1290--6-dev-docs-plone-org.netlify.app/backend/control-panels.html

Is this the page you want to link to?

https://training.plone.org/5/mastering-plone/registry.html#vocabularies-registry-settings-and-control-panels

Reply:
This only happens because the deployment of the training is currently broken because of links that are supposedly broken (they are not). I'll fix that.

Reply:
Fixed the training deploy. Now these links should start working.

Reply:
The link is still broken: https://deploy-preview-1290--6-dev-docs-plone-org.netlify.app/backend/control-panels.html

Reply:
I retried a Netlfiy preview deploy, as it hadn't been updated in a couple of days. The link now renders correctly. Resolving.